### PR TITLE
OKTA-530037 Updated screenshots and text for new hookdeck UI

### PR DESCRIPTION
**Pull request by danielmaharry-okta**
## Description:
- **What's changed?** : Updated screenshots and text for the "Using Hookdeck with Event Hooks" guide to reflect new Hookdeck UI.

- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-530037](https://oktainc.atlassian.net/browse/OKTA-530037)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/guides/event-hook-hookdeck/main/index.md](https://github.com/okta/okta-developer-docs/blob/a522e32b3ccea4edea1154fe435c22e847448976/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fevent-hook-hookdeck%2Fmain%2Findex.md) | 72 | 114 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/4fc00d05-53f6-4949-9aeb-f9287c58a7c8) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/guides/event-hook-hookdeck/main/nodejs/sample-app.md](https://github.com/okta/okta-developer-docs/blob/a522e32b3ccea4edea1154fe435c22e847448976/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fevent-hook-hookdeck%2Fmain%2Fnodejs%2Fsample-app.md) | 63 | 19 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/913bd01e-bb93-4b57-ba80-4f1f93faa142) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 2 of 2 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.e25d078d-4376-449f-81e3-7bbc998ad340?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

File: packages/@okta/vuepress-site/docs/guides/event-hook-hookdeck/main/index.md [hunk 1]
```diff
@@ -197,11 +198,11 @@ To run a test of your event hook:
 
 1. Review your Hookdeck terminal output for a line item reference to the specific call and a unique dashboard URL to the details on the call.
 
-1. Review the unique dashboard URL (for example, `https://dashboard.hookdeck.com/cli/events/evt_x3H5QdAL2JGvxMbvqcvwK8UF`). Scroll to the **Body** section and open the **"Root"** > **"Data"** sections. Your new user's name and username appear under the **"target"** property.
+1. Review the unique dashboard URL (for example, `https://dashboard.hookdeck.com/cli/events/evt_x3H5QdAL2JGvxMbvqcvwK8UF`). Scroll to the **Body** section and search for the **target** property. Your new user's name and id appear there.
```
Comment (brianduffield-okta) [inline packages/@okta/vuepress-site/docs/guides/event-hook-hookdeck/main/index.md:201]:
```suggestion
1. Review the unique dashboard URL (for example, `https://dashboard.hookdeck.com/cli/events/evt_x3H5QdAL2JGvxMbvqcvwK8UF`). Scroll to the **Body** section and search for the **target** property. Your new user's name and ID appear there.
```

**Review by brianduffield-okta — APPROVED** [review]
LGTM, Dan! One small hit and this image update suggestion: `hookdeck-event-hook-body.png` to remove your test org from the `source` url.

File: packages/@okta/vuepress-site/docs/guides/event-hook-hookdeck/main/index.md (hunk 1 above)
Comment (danielmaharry-okta) [reply packages/@okta/vuepress-site/docs/guides/event-hook-hookdeck/main/index.md:201]: fixed

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/guides/event-hook-hookdeck/main/index.md](https://github.com/okta/okta-developer-docs/blob/e811a79f6534d1ecfef7f55c51abcc12f310b821/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fevent-hook-hookdeck%2Fmain%2Findex.md) | 72 | 114 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/8deca229-b0fe-4fad-afe1-9e9df8a06880) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/guides/event-hook-hookdeck/main/nodejs/sample-app.md](https://github.com/okta/okta-developer-docs/blob/e811a79f6534d1ecfef7f55c51abcc12f310b821/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fevent-hook-hookdeck%2Fmain%2Fnodejs%2Fsample-app.md) | 63 | 19 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/148e3248-e1ed-41e6-91fe-bf62a6bafc3b) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 2 of 2 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.6066d8e2-b73d-4637-a099-0010a366e856?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

**Pull request merged**